server/src/main/java/uk/ac/cam/cl/echo/extrusionfinder/server/preprocessor/ColorSpaceConversion.java

```diff
@@ -0,0 +1,180 @@
+package uk.ac.cam.echo.extrusionfinder.preprocessor;
+
+import java.lang.Math;
+
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.LabImageData;
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.RGBImageData;
+import uk.ac.cam.cl.echo.extrusionfinder.server.imagedata.XYZImageData;
+
+/**
+ * Collection of static colorspace conversion methods.
+ * Includes pairwise conversions between RGB, XYZ and L*a*b*.
+ */
+public class ColorSpaceConversion {
+
+    // The CIE XYZ tristimulus values of the reference white point for D6500
+    // wikipedia.org/wiki/Illuminant_D65#Definition
```

Comment:
I can't really comment the fields since they're pretty much "arbitrary" constants unless I understood more of the maths.